pymc/tests/distributions/test_shape_utils.py

```diff
@@ -573,7 +574,7 @@ def test_change_rv_size_default_update():
 def test_change_specify_shape_size_univariate():
     with aesara.config.change_flags(mode=Mode("py")):
         s1, s2 = at.iscalars("s1", "s2")
-        x = at.random.normal(size=(s1, s2))
+        x = pymc_normal(size=(s1, s2))
```

Comment:
We don't need to change the object used in tests that are not related to the docstrings